### DISCUSSION TITLE
Vercel Deployments having issues with Next-Auth (Round 2)

### DISCUSSION THREAD

**Discussion by gera-cameron** [vercel/community#2156] [Help] [answered]
### Summary

This is a repeat of https://github.com/orgs/vercel/discussions/1493, but this issue has reappeared.

**Comment by coltonevansid** [comment]
We started seeing same issue this morning. Any new builds on Vercel cause our Auth to fail. Any builds prior to this morning still work.

**Reply by coltonevansid** [reply]
Same issue here: https://github.com/orgs/vercel/discussions/2169

**Comment by pdparchitect** [comment]
I really hope that at least this time we get an idea of what has really caused this issue. A postmortem would help.

**Comment by Cl3arglass** [comment, marked as answer]
Hello, we reverted a change that we believe should fix the problem. Please try and redeploy your project, and let us know if the issue persists: https://vercel.com/help

**Reply by gera-cameron** [reply]
@Cl3arglass would you be able to share what the problem was? This is the second time this has happened and it is a cause for concern that would lead me and my team away from Vercel. We would appreciate your transparency, thank you!
